test/pre-reqs.md

```diff
@@ -16,8 +14,12 @@ The following pre-requisites are required:
     curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
     export PATH=%PWD/ocm:$PATH
     ```
+    At the end of the install make sure that the OCM CLI, clusteradm, is in your path.
 
 - helm - to deploy the kubestellar and kubeflex charts
 - kind - to create a few small kubernetes clusters
-
+- kubectl - to access the kubernetes clusters
+- go version 1.20 or higher - to build kubestellar
+- make - to build kubestellar and create the kubestellar image 
+- ko - to create the kubestellar image
```

Comment:
This is good for now, we may want to include check for presence of ko and install in the Makefile, the same way kustomize is installed if missing.